tools/lint

```diff
@@ -47,9 +47,12 @@ def run():
     parser.add_option('--full',
                       action='store_true',
                       help='Check some things we typically ignore')
-    parser.add_option('--pep8',
-                      action='store_true',
-                      help='Run the pep8 checker')
+    parser.add_option('--py',
+                      action="store_true",
+                      help='Don\'t run expensive Javascript checks')
+    parser.add_option('--js',
+                      action="store_true",
+                      help='Don\'t run expensive Python checks')
```

Comment:
@timabbott yeah, just noticed that I didn't grep for any uses of --pep8 :). On the other hand, there seems to be no place that runs only `tools/lint`, so there should be no need to amend any --js to existing calls.